Products/ZenUI3/browser/resources/js/zenoss/network.js

```diff
@@ -152,6 +152,13 @@ function treeselectionchange(sm, newnodes) {
     }
     var newnode = newnodes[0];
     var uid = newnode.data.uid;
+    if (uid.match(/:backward:|:forward:/)) {
```

Comment:
Use separate fields (updating the model and what the server sends) instead of overloading the uid field.